File: javascript/src/fizzbuzz.js
module.exports = function(input) {
    let output = ''
    if(input % 3 === 0) 
        output += 'Fizz'
    if(input % 5 === 0) 
        output += 'Buzz'
    if(input % 99 === 0) output += 'Strange'
    if(input % 101 === 0) output += 'Even Stranger'


    return output.length > 0 ? output : input
}

File: java/src/main/java/fizzbuzz/FizzBuzz.java
package fizzbuzz;

public class FizzBuzz {

    public String fizzbuzz(int input){
        String output = "";
        if( input % 3 == 0 ) output += "Fizz";
        if( input % 5 == 0 ) output += "Buzz";

        return output.length() > 0 ? output : Integer.toString(input);
    }

}
